x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -168,83 +192,124 @@ describe('index table', () => {
     server.restore();
   });
 
-  test('should change pages when a pagination link is clicked on', () => {
+  test('should change pages when a pagination link is clicked on', async () => {
     const rendered = mountWithIntl(component);
+    await runAllPromises();
+    rendered.update();
+
     snapshot(namesText(rendered));
+
     const pagingButtons = rendered.find('.euiPaginationButton');
     pagingButtons.at(2).simulate('click');
-    rendered.update();
     snapshot(namesText(rendered));
   });
-  test('should show more when per page value is increased', () => {
+
+  test('should show more when per page value is increased', async () => {
```

Comment:
Are we testing the EUI table component here?

Reply:
The indices table is implemented using individual `EuiTable`, `EuiTablePagination`, `EuiTableRowCell`, etc components. So unfortunately, no, this isn't testing EUI. It's testing our table implementation which EUI now provides an abstraction over with `EuiBasicTable`. More tech debt! 🎉